actions/README.md

```diff
@@ -0,0 +1,33 @@
+# slsa-verifier-installer GitHub Action
+
+This action installs the SLSA verifier so you can verify SLSA provenance in your workflows.
+
+For more information about `slsa-verifier`, refer to the [its documentation](https://github.com/slsa-framework/slsa-verifier#verification-of-provenance).
+
+For more information about SLSA in general, see https://slsa.dev.
+
+## Usage
+
+This action requires using GitHub-provided Linux runners.
+
+Add the following snippet to your Github workflow YAML file:
+
+```yaml
+uses: slsa-framework/slsa-verifier-installer@main
+with:
+    verifier-release: v1.3.0 # optional
+    install-dir: /tmp/slsa-verifier # optional
+    use-sudo: false # optional
+```
+
+For a full example workflow, see `.github/workflows/test.yml`.
+
+### Optional Inputs
+
+| Input | Description |
+| --- | --- |
+| `verifier-release` | `slsa-verifier` version to use. Default value is `v1.3.0`. |
+| `install-dir` | Where to install `slsa-verifier` binary. Default value is `$HOME/slsa-verifier`. |
+| `use-sudo` | Whether the install location requires `sudo`. Default value is `false`.
```

Comment:
Do we really need this? I'm pretty user actions just run as root in their environment anyway.

Reply:
I'm also in favor of not exposing such option.

Reply:
Done

Reply:
Done